spec/models/game_spec.rb

```diff
@@ -1,11 +1,77 @@
 require 'rails_helper'
 
 RSpec.describe Game, type: :model do
-  describe 'populate game' do
-    it 'should populate the game with pieces' do
-      game = FactoryGirl.build(:game)
+  describe '#populate game' do
+    it 'should start with 32 pieces' do
+      game = Game.create
       game.populate_game!
-      expect(game.pieces.count).to eq(32)
+      array = game.pieces.map { |p| p }
+      expect(array.count).to eq(32)
```

Comment:
was `expect(game.pieces.count).to eq(32)` not working here?

I'd go with ^ over converting to an array. Same result, less code.

Also, I think active record relations have a `to_array` method. So in future, could use that instead of `map`